src/renderer/api/terminal-api.ts

```diff
@@ -144,14 +144,6 @@ export class TerminalApi extends WebSocketApi<TerminalEvents> {
     this.socket.binaryType = "arraybuffer";
   }
 
-  destroy() {
```

Comment:
This is not needed, we kill the underlying process on normal socket close anyway

Reply:
Are you 100% sure this won't affect behaviour on all platforms? If this isn't related to the fix I'd leave it in.

Reply:
Okay, I'll make a different PR to remove this.